Installations-HowTos/How-To-Install-Spark-On-MACOS.md

```diff
@@ -1,22 +1,21 @@
 
 # Step-1 - Setup Java JDK
 
-Install java JDK on your machine 
+Install java JDK on your machine (https://www.oracle.com/java/technologies/downloads/#jdk21-mac)
 
 
 java -version
 
-java version "1.8.0_211"
-Java(TM) SE Runtime Environment (build 1.8.0_211-b12)
-Java HotSpot(TM) 64-Bit Server VM (build 25.211-b12, mixed mode)
-
+java version "21.0.1" 2023-10-17 LTS

```

Comment:
Let's add an important notes to not use Java 16.

Important NOTE: Do NOT Use Java 16. Other Java version existing on your system should work fine and you don't have to reinstall

Reply:
@charlie-ph completely agree, added the same